src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -378,8 +378,8 @@ protected function mergeWithExisting($entitiesByModule)
             // Merge
             if ($existingMessages) {
                 $entitiesByModule[$module] = array_merge(
-                    $existingMessages,
-                    $messages
+                    $messages,
```

Comment:
Correct - you don't want to be merging then.

If I call it /dev/tasks/i18nTextCollectorTask/?module=themes&locale=ja_JP I expect it to keep my existing ja_JP translations and add any new messages with their defaults into the file. Currently it'd lose all my ja_JP translations as they'd be overwritten by the defaults.